src/Sylius/Behat/Context/Api/Shop/LoginContext.php

```diff
@@ -268,7 +269,10 @@ public function iShouldNotBeAbleToChangeMyPasswordAgainWithTheSameToken(): void
     {
         $this->client->executeCustomRequest($this->request);
 
-        Assert::same($this->client->getLastResponse()->getStatusCode(), 404);
+        // token is removed when used
+        Assert::same($this->client->getLastResponse()->getStatusCode(), 500);
+        $message = $this->responseChecker->getError($this->client->getLastResponse());
+        Assert::startsWith($message, 'No user found with reset token: ');
```

Comment:
Do you mind creating additional PR with that change? I think we might want to remove occurrence of 500 in our API 😜

Reply:
Yes, I thought about replacing the Assert throwing the 500 by a NotFoundException. Is that the way to do it ?

Reply:
Yea, that's it :)

Reply:
Definitely a separate PR would be welcomed, let's focus on api prefix in this one 🖖

Reply:
@Zales0123 @Ferror #13902 created to fix it.

Reply:
Let's merge #13902 at a separate pace. May I have a silly question? Why did you change it? I mean, I understand that 404 will be returned from this endpoint only after #13902 will be merged, but yet the current master build is green without this or #13902 change. Why?

Reply:
@lchrusciel the response was a 404 because the route was not found :

```json
{
    "code":404,
    "message":"No route found for \u0022PATCH http:\/\/localhost\/api\/v2\/shop\/reset-password-requests\/api\/v2\/shop\/reset-password-requests\/itotallyforgotmypassword\u0022 (from \u0022http:\/\/localhost\/api\/v2\/shop\/reset-password-requests\/itotallyforgotmypassword\u0022)"
}
```

My changes is altering the route from `api/v2/...` to `/api/v2/...` that's why the real error is showing.